app/components/Ui/badge_component.rb

```diff
@@ -6,11 +6,12 @@ def initialize(color:)
   def color_classes
     case color
     when 'yellow'
-      'bg-yellow-50 text-yellow-700 ring-yellow-600/20'
+      'bg-yellow-50 text-yellow-700 ring-yellow-600/20 ' \
+      'dark:bg-yellow-400/10 dark:text-yellow-500 dark:ring-yellow-400/20'
```

Comment:
Is the line split/backslash intentional for "yellow"?

Reply:
Yeah its intentional, the linter was complaining about line length. Back slash is a way of concatenating multiline strings in Ruby.

Reply:
Ah, gotcha. Looks great to me then!